diagram.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/golang/freetype"
 	"github.com/golang/freetype/truetype"
 
-	"github.com/akavel/ditaa/embd"
-	"github.com/akavel/ditaa/fontmeasure"
-	"github.com/akavel/ditaa/graphical"
+	"./embd"
+	"./fontmeasure"
+	"./graphical"
```

Comment:
After reading a bit more, I think my error was in not checking out the repo to ``$GOPATH/src/github.com/akavel/ditaa`` - I assumed $GOPATH to be more like a ``.node_modules``. So I guess you're right and local paths are really not the go way.